packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -172,17 +223,21 @@ export function createResponseState(
       bootstrapChunks.push(endAsyncScript);
     }
   }
+  const instructionWriterState: DefaultInstructionWriterState = {
+    startInlineScript: inlineScriptWithNonce,
+    sentCompleteSegmentFunction: false,
+    sentCompleteBoundaryFunction: false,
+    sentClientRenderFunction: false,
+  };
   return {
     bootstrapChunks: bootstrapChunks,
-    startInlineScript: inlineScriptWithNonce,
-    placeholderPrefix: stringToPrecomputedChunk(idPrefix + 'P:'),
-    segmentPrefix: stringToPrecomputedChunk(idPrefix + 'S:'),
+    placeholderPrefix: idPrefix + 'P:',
```

Comment:
These were changed to strings, since we want to format IDs before passing them to `InstructionWriters`. I had considered concatenating or building up PrecomputedChunks instead, but not sure if (1) it would improve perf (we'd probably had to allocate + write to intermediate buffers) and (2) if it's needed.

If the perf regression is too much, I can try to figure out a better solution